webapp/channels/src/utils/performance_telemetry/reporter.ts

```diff
@@ -78,13 +85,17 @@ export default class PerformanceReporter {
     protected reportPeriodBase = 60 * 1000;
     protected reportPeriodJitter = 15 * 1000;
 
-    constructor(client: Client4, store: Store<GlobalState>) {
+    constructor(client: Client4, store: Store<GlobalState>, desktopAPI: DesktopAppAPI) {
```

Comment:
any particular reason why we are sending metrics over from desktop to web via desktopapi when we can do this in desktop app itself

Reply:
i havent found the reason why we needed to transmit back from desktop to webapp in the spec. I will comment over there

Reply:
Commented on the document.